schemas/expconf/v0/log-policy.json

```diff
@@ -3,22 +3,32 @@
     "$id": "http://determined.ai/schemas/expconf/v0/log-policy.json",
     "title": "LogPolicy",
     "additionalProperties": false,
-    "required": [
-        "pattern",
```

Comment:
Isn't pattern still required?

Reply:
I make it not required so user can override the default polices.
```
log_policies:
  - name: CUDA OOM
  - name: ECC Error
```